packages/pong-engine/src/game.ts

```diff
@@ -58,3 +81,21 @@ export const runGameFrame = (
   }
   return reducer(state, { type: 'move', payload: { deltaTime } });
 };
+
+export const runGameMultiplayerFrame = (
+  deltaTime: number,
+  state: GameState,
+): GameState => {
+  const { ball } = state;
+
+  if (losePointMultiplayer(ball)) {
```

Comment:
IMHO multiplayer should be the non suffixed default. We shouldn't have to say `losePointMultiplayer` vs `losePoint`. It should rather be `losePoint` vs `losePointSinglePlayer`, the first one not defaulted.

Reply:
This is done like this to have retro compatibility with the game training game.

We could change it but making one be the multiplayer or the other being the training wouldn’t be much different, the thing is to have clear which one is which